src/scss/_vars.scss

```diff
@@ -1,6 +1,32 @@
 // prefix
 $px: seldon;
 
+@mixin font-size-vars--sm {
+  --font-size-micro: 0.25rem;
+  --font-size-xsm: 0.5rem;
+  --font-size-sm: 0.75rem;
+  --font-size-md: 1rem;
+  --font-size-lg: 1.25rem;
+  --font-size-xl: 1.5rem;
+  --font-size-xxl: 1.75rem;
+  --font-size-xxxl: 2rem;
+}
+
+@mixin font-size-vars--md {
+  @include font-size-vars--sm;
+}
+
+@mixin font-size-vars--lg {
+  --font-size-micro: 0.5rem;
+  --font-size-xsm: 0.75rem;
+  --font-size-sm: 1rem;
+  --font-size-md: 1.25rem;
+  --font-size-lg: 1.5rem;
+  --font-size-xl: 1.75rem;
+  --font-size-xxl: 2rem;
+  --font-size-xxxl: 2.25rem;
+}
+
```

Comment:
Can you help me understand the value of this abstraction here? Are we going to use these mixins in other areas? I think there may be value in seeing the size changes in the media query for quick reference? If we do not re-use the mixin I feel we lose the visibility without any other benefit.